include/aie/Dialect/AIE/Transforms/AIEAssignBufferDescriptorIDs.h

```diff
@@ -11,27 +11,8 @@
 #ifndef AIE_ASSIGN_BUFFER_DESCRIPTOR_IDS_H
 #define AIE_ASSIGN_BUFFER_DESCRIPTOR_IDS_H
 
-#include <optional>
+#include "aie/Dialect/AIE/IR/AIETargetModel.h"
 
-#include "aie/Dialect/AIE/IR/AIEDialect.h"
-#include "aie/Dialect/AIE/Transforms/AIEAssignBufferDescriptorIDs.h"
-#include "aie/Dialect/AIE/Transforms/AIEPasses.h"
-
-#include "mlir/Pass/Pass.h"
-
-using namespace mlir;
-using namespace xilinx;
-using namespace xilinx::AIE;
-
-#include "aie/Dialect/AIE/IR/AIEDialect.h"
-#include "aie/Dialect/AIE/Transforms/AIEPasses.h"
-
-#include "mlir/Pass/Pass.h"
-
-#define DEBUG_TYPE "aie-assign-bd-ids"
-
-using namespace mlir;
-using namespace xilinx;
 using namespace xilinx::AIE;
```

Comment:
Thanks for all the seasonal cleaning!
Even the last one is too much for my taste, but it would be probably too extreme for now. :smiley:

Reply:
> Thanks for all the seasonal cleaning! Even the last one is too much for my taste, but it would be probably too extreme for now. 😃

yeah it's a slippery slope. Before you know it you're "cleaning" every file in the repo.